run_sync

```diff
@@ -3,4 +3,10 @@
 mkdir -p /mnt/sync/folders
 mkdir -p /mnt/sync/config
 
+if [ -f /mnt/sync/sync.conf ]; then
+    rm -f /etc/sync.conf;
```

Comment:
No need to remove - you can rename it to something like /etc/sync/default.config and leave it in the container